drawing.js

```diff
@@ -38,19 +38,19 @@ function draw () {
   if (state < STATE_AUDIO) {
     viz.background(bgColor)
 
-    viz.textSize(32)
-    viz.fill(150)
-    viz.text('Preparing stuff', WIDTH / 2 - 100, HEIGHT / 2)
+    viz.textSize(20)
+    viz.fill(300)
```

Comment:
`fill()` takes color as a parameter. 300 means gray value = 300 which is out of bounds (0-255 being the permissible range). check this out - https://p5js.org/learn/color.html